src/System Application/App/Document Processor/src/PDFHelper.Codeunit.al

```diff
@@ -0,0 +1,33 @@
+
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace System.IO;
+
+using System.Utilities;
+
+/// <summary>
+/// Codeunit that provides helper functions for PDF processing.
+/// </summary>
+codeunit 3110 "PDF Helper"
```

Comment:
No helpers, helpers means this is a hack. We introduced debt here.
Management/Mgt. means less hacks, we do not have a clear purpose. 

I would name the file PDF Document

Reply:
Resolved.